### PR TITLE
Support sending raw IMU data to a UART

### PR DESCRIPTION
**Pull request by tridge**
This allows unfiltered IMU data to be sent at the main loop rate to a UART. This is for supporting external visual odomotry systems that need to augment image data with the autopilots IMU data

was used for this ANU research project: https://github.com/STR-ANU/eqf_vio

### CODE REVIEW

**Comment by peterbarker** [comment]
This looks like cooked data?

**Comment by tridge** [comment]
> This looks like cooked data?

It is loop rate data, not filtered, same data the EKF gets, which is exactly what we want, as we're running an external state estimator

**Comment by peterbarker** [comment]
@tridge my point is that we use the word "raw" to mean something different in the IMU code.  Perhaps `Support sending loop-rate IMU data...`

**Comment by peterbarker** [comment]
My "raw" vs "cooked" thing has been addressed.

Needs a rebase, 'though.  Almost certainly the serial protocol number's been used.

File: libraries/AP_SerialManager/AP_SerialManager.cpp [hunk 1]
```diff
@@ -482,6 +483,16 @@ void AP_SerialManager::init()
                     state[i].uart->set_flow_control(AP_HAL::UARTDriver::FLOW_CONTROL_DISABLE);
                     break;
 
+                case SerialProtocol_IMUOUT:
+#if !HAL_MINIMIZE_FEATURES && !defined(HAL_BUILD_AP_PERIPH)
```
Comment (magicrub) [inline libraries/AP_SerialManager/AP_SerialManager.cpp:487]: Why is this !defined(HAL_BUILD_AP_PERIPH)?

**Comment by tridge** [comment]
rebased

**Comment by tridge** [comment]
@peterbarker it is unfiltered

**Review by peterbarker — APPROVED** [review]

**Pull request merged**